satellite/attribution/db.go

```diff
@@ -9,8 +9,12 @@ import (
 	"time"
 
 	"github.com/skyrings/skyring-common/tools/uuid"
+	"github.com/zeebo/errs"
 )
 
+// ErrNoAttributionFound is returned if a node does not exist in database
+var ErrNoAttributionFound = errs.Class("bucket not attributed")
```

Comment:
variable name should reflect error message e.g. `ErrBucketNotAttributed  = errs.Class("bucket not attributed")`. Also I think comment is copy pasted

Reply:
oops... !!!fixed